File: src/server/TypeORM/src/entity/work_user.ts
import {Entity, PrimaryGeneratedColumn, Column} from "typeorm";

@Entity()
export class work_user {

    @PrimaryGeneratedColumn()
    work_user_id: string;

    @Column()
    firstname: string;

    @Column()
    lastname: string;

    @Column()
    email: string;

}

File: fg_metrics/VSTS/VSTSComms/VSTSComms/EntitiesMySQL/WorkUser.cs
using System;
using System.Collections.Generic;

namespace VSTSComms.EntitiesMySQL
{
    public partial class WorkUser
    {
        public WorkUser()
        {
            WorkTeamMember = new HashSet<WorkTeamMember>();
        }

        public string WorkUserId { get; set; }
        public string Firstname { get; set; }
        public string Lastname { get; set; }
        public string Email { get; set; }
        public bool? Active { get; set; }

        public ICollection<WorkTeamMember> WorkTeamMember { get; set; }
    }
}
